akka-docs/rst/scala/code/docs/http/scaladsl/SprayJsonExampleSpec.scala

```diff
@@ -47,4 +49,48 @@ class SprayJsonExampleSpec extends WordSpec with Matchers {
       //#
     }
   }
+
+  "second-spray-json-example" in compileOnlySpec {
+    import akka.actor.ActorSystem
+    import akka.stream.ActorMaterializer
+    import akka.Done
+    import akka.http.scaladsl.server.Route
+    import akka.http.scaladsl.server.Directives._
+    import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+    import spray.json.DefaultJsonProtocol._
```

Comment:
Since we have all imports enumerated, I think we should also mention that this example requires spray-json library and marshaling support artifacts to run.